setup.rst

```diff
@@ -341,6 +341,14 @@ Now you can install the build dependencies via ``apt``::
 If that package is not available for your system, try reducing the minor
 version until you find a package that is available.
 
+If you want to build all modules, install these packages in the versions you
+prefer, changing the version name as above::
```

Comment:
I think this can be combined with the above point, as I'm pretty sure the explicit package names cover everything that would be installed by `build-dep`.  The above point should also be updated to a more recent version (or the minor version removed entirely, and the suggestion amended to suggest adding a minor version if `python3` doesn't cut it).

Reply:
I have removed the minor version from the Python package for which the command installs the build dependencies.